core/templates/dev/head/pages/exploration_editor/RouterServices.js

```diff
@@ -167,15 +167,15 @@ oppia.factory('routerService', [
         if (_tabs.active === MAIN_TAB) {
           $('.oppia-editor-cards-container').fadeOut(function() {
             _actuallyNavigate(SLUG_GUI, stateName);
-            // jscs:disable maximumLineLength
+            // eslint-disable max-len
```

Comment:
eslint max-len has an ignore-urls option. If that is used, do you still need this pragma?